coprocess_grpc_test.go

```diff
@@ -204,26 +208,28 @@ func loadTestGRPCAPIs() {
 	})
 }
 
-func startTykWithGRPC() (*tykTestServer, *grpc.Server) {
+func startTykWithGRPC(grpcPort int) (*tykTestServer, *grpc.Server) {
 	// Setup the gRPC server:
-	listener, _ := net.Listen("tcp", grpcListenAddr)
+	listener, _ := net.Listen("tcp", fmt.Sprintf(":%d", grpcPort))
```

Comment:
It's better to use `strconv.Itoa`. Sprintf will use the package reflect, and it will allocate one more object.